factory/reader.js

```diff
@@ -35,7 +70,7 @@ function showContent(url) {
   requestContent(url).then(function (body) {
     var content = parseContent(body);
     printArticle(content);
-  }, function () {
+  }, function (e) {
```

Comment:
Any reason event was passed here?..because it's not used within the body of that function.